blueoil/generate_lmnet_config.py

```diff
@@ -97,11 +98,11 @@ def _load_yaml(blueoil_config_filename):
     Returns:
         blueoil_config(dict): dict of blueoil config.
     """
-    if not os.path.exists(blueoil_config_filename):
+    if not gfile.Exists(blueoil_config_filename):
         FileNotFoundError("File not found: {}".format(blueoil_config_filename))
 
-    with open(blueoil_config_filename, "r") as f:
-        blueoil_config = yaml.load(f)
+    with gfile.GFile(blueoil_config_filename, "r") as f:
+        blueoil_config = yaml.load(f, Loader=yaml.Loader)
```

Comment:
why `Loader=yaml.Loader` is needed?

Reply:
@iizukak Thanks for your comment.
Because I wanted to get rid of this warning.
```
YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
```

And I followed the same way how to read yaml in lmnet.
https://github.com/blue-oil/blueoil/blob/60d18e041d5a6cc42293957d5a1155a2aa742b18/lmnet/lmnet/utils/config.py#L247-L249

Reply:
Thanks

Reply:
`yaml.SafeLoader` is recommended by PyYAML, why do you use `yaml.Loader` ?

Reply:
I used `yaml.Loader` because this loader is the default one in PyYAML==4.2 and  I didn't want to change the behavior of `yaml.load()` from the current code.

But in this case, given yaml files should have a simple yaml format. I agree with using `yaml.SafeLoader` 👍